hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpConfiguration.java

```diff
@@ -58,6 +60,9 @@
     @Autowired
     private ConnectionFactory rabbitConnectionFactory;
 
+    @Autowired
+    private ErrorHandler errorHandler;
```

Comment:
@schabdo this was my first thought when I saw the row. It's difficult to understand and confusing.